app/src/main/java/http_server/ResponseHandler.java

```diff
@@ -0,0 +1,38 @@
+package http_server;
+
+public class ResponseHandler {
+  Response response = new Response();
```

Comment:
Just talking about naming things, this class seems to build a response.

Reply:
Oh true! good point!